src/partition/FileUtils.h

```diff
@@ -59,6 +65,35 @@ class InputFile {
     off_t _sz;
 };
 
+class InputFileArrow : public InputFile {
+public:
+    explicit InputFileArrow(boost::filesystem::path const &path, off_t blockSize);
```

Comment:
`explicit` only makes sense for a single parameter.

Reply:
Incorrect formatting in the parameter declaration:
```
boost::filesystem::path const &path
```
Has to be:
```
boost::filesystem::path const& path
```

Reply:
I turned all the const &path into const& path, but the clang formatting command turned most of them back into const &path.